source/dub/commandline.d

```diff
@@ -257,6 +257,7 @@ struct CommonOptions {
 	/// Parses all common options and stores the result in the struct instance.
 	void prepare(CommandArgs args)
 	{
+		registry_urls = environment.get("DUB_REGISTRY", null).split(";");
```

Comment:
I think this should rather be done in the `Dub` class, so that the command line module ideally performs pure command line parsing (I realize that it already reads `DFLAGS`, which should really be refactored). A new mode for `SkipPackageSuppliers` *could* then also be added to enable choosing between using only those specified on the command line, or also those coming from the environment.